### DISCUSSION TITLE
Can I connect to SuperSet using ssh?

### DISCUSSION THREAD

**Discussion by Asfering** [apache/superset#19862] [Q&A / Help]
Hello everyone!
My database (Greenplum => Postgresql) is on a virtual machine, ubuntu 18.04. 
I have configured the connection via network adapters in such a way that I can connect to the database from Windows in pgAdmin 4 using SSH
PgAdmin4:
![image](https://user-images.githubusercontent.com/63109582/164759677-c7006d1c-c1dd-4921-a769-9d53a70949d7.png)
![image](https://user-images.githubusercontent.com/63109582/164759721-b20cc8d4-2c58-40dd-84ca-3d76259b5128.png)
This connection is correct, I can use DB in Windows.

And now I want to use this DB in superset, which staying on ubuntu application on windows:
![image](https://user-images.githubusercontent.com/63109582/164760155-883e1fad-f736-4c70-b1fb-695b22e5d59e.png)
What do I need to write here? Is it even possible to use SSH connection in Superset? Maybe I need to write a connection string somehow?
![image](https://user-images.githubusercontent.com/63109582/164760317-b3d22405-62bc-4ef9-b07b-5df839b8680f.png)

I hope I have made myself clear and you understand me. All the best.

**Comment by mohammeddib10** [comment]
@Asfering Any news about this subject? I need to do the same thing.
